src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -589,8 +592,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="_AddPublishWasmBootJsonToStaticWebAssets" DependsOnTargets="GeneratePublishWasmBootJson">
     <ItemGroup>
       <_PublishWasmBootJson
-        Include="$(IntermediateOutputPath)blazor.publish.boot.json"
-        RelativePath="_framework/blazor.boot.json" />
+        Include="$(IntermediateOutputPath)$(_WasmPublishBootConfigFileName)"
```

Comment:
this will become `publish.blazor.boot.json` is that OK ?

Reply:
I have googled around, and nobody is reading the intermediate file